api/scripts/wait-for-postgres.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
This script is used in our service startup (docker-compose.yaml) to force the API & database migration containers to wait until postgres is available before running.